sema/ro/creator/__main__.py

```
@@ -0,0 +1,155 @@
+    root = Path(root)
+    rocyml = Path("roc-me.yml")
+    if not root.exists():
+        raise FileNotFoundError(f"root path {root} does not exist")
+
+    if root.is_dir():
+        # find roc-*yml file -- warn if more then one, use if only one
+        rocfiles: list[Path] = getMatchingGlobPaths(root, "roc-*.yml")
+        if len(rocfiles) > 1:
+            raise ValueError(
+                f"multiple roc yml files found in {root}, cannot proceed"
+            )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)